frontend/src/metabase/visualizations/components/settings/ChartSettingOrderedSimple.tsx

```diff
@@ -0,0 +1,73 @@
+import { updateIn } from "icepick";
+import React from "react";
+import { t } from "ttag";
+
+import { ChartSettingOrderedItems } from "./ChartSettingOrderedItems";
+
+import {
+  ChartSettingMessage,
+  ChartSettingOrderedSimpleRoot,
+} from "./ChartSettingOrderedSimple.styled";
+
+interface SortableItem {
+  enabled: boolean;
+  originalIndex: number;
+  name: string;
+}
+
+interface ChartSettingOrderedSimpleProps {
+  onChange: (rows: SortableItem[]) => void;
+  items: SortableItem[];
+  value: SortableItem[];
+}
+
+export const ChartSettingOrderedSimple = ({
```

Comment:
I realized while working on this that we could drive the reordering with the same component, but it had `Row` plastered everywhere, so I thought of renaming it and it's properties to something more generic might be a good idea. `<ChartSettingOrderedItems />` houses the actual draggable elements. `<ChartSettingOrderedSimple />` takes care of the logic for what happens when we interact with the items.

Reply:
yeah that makes sense, I just thought there might be a better name than `ThingSimple` but since I couldn't think of that myself, it's safe to leave it as is.